docs/guide/generating_key_pair.rst

```diff
@@ -0,0 +1,46 @@
+Generating a Key pair
+=====================
+
+Key pair
+--------
+
+The Key pair is a pair of private and public keys. The private key is used to sign transactions, and the public key is used to verify the signature.
+In the starknet.py you need to use the :meth:`~starknet_py.net.signer.key_pair.KeyPair` class to be able to create an :meth:`~starknet_py.net.account.account.Account` and :meth:`~starknet_py.net.signer.stark_curve_signer.StarkCurveSigner` object.
+
+Generating random key pair
+--------------------------
+
+Method :meth:`~starknet_py.net.signer.key_pair.KeyPair.generate` generate cryptographically strong pseudo-random numbers
```

Comment:
This method isn't linked in the generated docs. I don't know why. Wdyt? @franciszekjob

Reply:
offline discussion: Looks like using ``:meth:`~starknet_py.net.signer.stark_curve_signer.KeyPair.generate`` resolves the issue, this is very strange though.